lib/vnet/Cargo.toml

```diff
@@ -12,6 +12,5 @@ wasmer-vfs = { path = "../vfs", version = "=3.0.0-rc.2", default-features = fals
 bytes = "1"
 
 [features]
-default = ["mem_fs"]
-mem_fs = ["wasmer-vfs/mem-fs"]
```

Comment:
mem-fs should still be a feature flag? Why did we removed it?

Reply:
Because almost every other filesystem depends on `mem_fs` in some way as a fallback. Plus, it's not a very heavy dependency, there's almost no point to having it disabled (saving two dependencies):

```
│   │       ├── slab v0.4.7
│   │       │   [build-dependencies]
│   │       │   └── autocfg v1.1.0
```